src/components/MessageList/styles.css

```
@@ -221,18 +221,30 @@ body.theme-light .default-background ~ * .message-list .message.transparent:hove
 /* System Message Styles - Now handled by MUI theme variant */
 
 .system-details-popover {
-  padding: 16px;
-  max-width: 400px;
-  max-height: 60vh;
-  overflow-y: auto;
+  --settings-chip-bg: rgba(255, 255, 255, 0.08);
+  /* Title font-size × line-height; shared so the tile count lines up with it. */
+  --settings-title-line: 1.365rem;
+  max-width: min(420px, calc(100vw - 32px));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)